editor/spatial_editor_gizmos.cpp

```diff
@@ -3584,10 +3584,10 @@ void CollisionPolygonSpatialGizmoPlugin::redraw(EditorSpatialGizmo *p_gizmo) {
 ////
 
 NavigationMeshSpatialGizmoPlugin::NavigationMeshSpatialGizmoPlugin() {
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge", Color(0.5, 1, 1)));
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge_disabled", Color(0.7, 0.7, 0.7)));
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_solid", Color(0.5, 1, 1, 0.4)));
-	create_material("navigation_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_solid_disabled", Color(0.7, 0.7, 0.7, 0.4)));
+	create_material("edge_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge", Color(0.5, 1, 1)));
+	create_material("edge_material_disabled", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_edge_disabled", Color(0.7, 0.7, 0.7)));
+	create_material("solid_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/navigation_solid", Color(0.5, 1, 1, 0.4)));
```

Comment:
I think those names are global, so it should still start with `navigation_`.

Reply:
See e.g.:
```
$ rg solid_material
editor/spatial_editor_gizmos.cpp
2176:   create_material("visibility_notifier_solid_material", gizmo_color);
2312:           Ref<Material> solid_material = get_material("visibility_notifier_solid_material", p_gizmo);
2313:           p_gizmo->add_solid_box(solid_material, aabb.get_size(), aabb.get_position() + aabb.get_size() / 2.0);
2325:   create_material("particles_solid_material", gizmo_color);
2465:           Ref<Material> solid_material = get_material("particles_solid_material", p_gizmo);
2466:           p_gizmo->add_solid_box(solid_material, aabb.get_size(), aabb.get_position() + aabb.get_size() / 2.0);
2483:   create_material("reflection_probe_solid_material", gizmo_color);
2642:           Ref<Material> solid_material = get_material("reflection_probe_solid_material", p_gizmo);
2643:           p_gizmo->add_solid_box(solid_material, probe->get_extents() * 2.0);
2659:   create_material("gi_probe_solid_material", gizmo_color);
2816:           Ref<Material> solid_material = get_material("gi_probe_solid_material", p_gizmo);
2817:           p_gizmo->add_solid_box(solid_material, aabb.get_size());
3607:   Ref<Material> solid_material = get_material("navigation_material", p_gizmo);
3608:   Ref<Material> solid_material_disabled = get_material("navigation_material", p_gizmo);
3689:   m->surface_set_material(0, navmesh->is_enabled() ? solid_material : solid_material_disabled);
```

Reply:
Updated.